app/vmselect/promql/transform.go

```diff
@@ -572,19 +572,21 @@ func vmrangeBucketsToLE(tss []*timeseries) []*timeseries {
 			})
 		}
 		xss = xssNew
-		for i := range xss[0].ts.Values {
-			count := float64(0)
-			for _, xs := range xss {
-				ts := xs.ts
-				v := ts.Values[i]
-				if !math.IsNaN(v) && v > 0 {
-					count += v
+		if len(xss) > 0 {
```

Comment:
It would be great to change this line to:

```go
if len(xss) == 0 {
  return rvs
}
```

Then the rest of lines shouldn't be changed.

Reply:
Do you mean 
```go
if len(xss) == 0 { 
  continue
}
```

if return rvs, the rest of timeseries can't be processd.

Reply:
Yes :) Then I'm going to merge this pull request as is and then adding tests from https://github.com/VictoriaMetrics/VictoriaMetrics/pull/2902 on top of it.